common/app/views/fragments/newHeader.scala.html

```diff
@@ -27,41 +39,44 @@
                     ))
                 </a>
 
-                <div class="new-header__cta-container hide-until-mobile">
+                <div class="new-header__top-bar hide-until-mobile">
                     @defining(
                         Edition(request).id.toLowerCase()
                     ) { editionId =>
                       @if(!page.metadata.hasSlimHeader) {
-                            <a class="header-cta-item header-cta-item--primary"
+                            <a class="top-bar__item top-bar__item--cta js-change-become-member-link"
                               data-link-name="nav2 : supporter-cta"
                               data-edition="@{editionId}"
                               href="@getContributionOrSupporterUrl(editionId)">
-                              <span class="header-cta-item__label">
+                              <span>
```

Comment:
what's this span for?

Reply:
Without the span I couldn't build the cta to look the way it does. The span around the text allows me to place the text in front of the pseudo element. Let me know if this is crazy talk!

Reply:
it might be handy to give it a class in that case, you can imagine the next developer seeing it and thinking it did nothing so ripping it out? 

it would also make the css a bit healthier (`.top-bar__item--cta--something` would be cleaner than `& > span`)

Reply:
That makes sense - I'll do that now.